### PR TITLE
Add initial profile readme

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Learn More!
+
+Jump straight into code by searching for your favorite lanague or framework in the repository list below, or these other great resources:
+
+📚 Read posts on the [Okta Developer Blog](https://developer.okta.com/blog/)! 
+
+💬 Ask a question on the [Okta Developer Forum](https://devforum.okta.com/)
+
+📺 Subscribe to [Okta Dev YouTube channel](https://www.youtube.com/c/oktadev) or on [Twitch](https://www.twitch.tv/oktadev)
```
Comment (mraible) [inline profile/README.md]:
```suggestion
📺 Subscribe to [OktaDev YouTube channel](https://www.youtube.com/c/oktadev) or on [Twitch](https://www.twitch.tv/oktadev)
```

**Review by mraible — APPROVED** [review]

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+![Okta Developer Logo](https://raw.githubusercontent.com/oktadev/.github/main/images/okta-dev-header.png)
+
+Want to learn about application security? Need to add auth into your project?  You have come to the right place!
+
+
+## Learn More!
+
+Jump straight into code by searching for your favorite lanague or framework in the repository list below, or these other great resources:
```
Comment (havi) [inline profile/README.md]:
```suggestion
Jump straight into code by searching for your favorite language or framework in the repository list below, or explore these other great resources:
```
fix typo, adjust copy.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+📚 Read posts on the [Okta Developer Blog](https://developer.okta.com/blog/)! 
+
+💬 Ask a question on the [Okta Developer Forum](https://devforum.okta.com/)
+
+📺 Subscribe to [OktaDev YouTube channel](https://www.youtube.com/c/oktadev) or on [Twitch](https://www.twitch.tv/oktadev)
```
Comment (havi) [inline profile/README.md]:
```suggestion
📺 Subscribe to the [OktaDev YouTube channel](https://www.youtube.com/c/oktadev) or find us live on [Twitch](https://www.twitch.tv/oktadev). 
```
copy tweaks

**Review by havi — COMMENTED** [review]
a couple of quick language/grammar tweaks

**Pull request merged**